backend/open_webui/utils/oauth.py

```
@@ -3,16 +3,17 @@
 import mimetypes
 import sys
 import uuid
-
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)